docs/Tutorials/Visualization.md

```diff
@@ -101,8 +101,8 @@ cause the program to exit after the information has been dumped. You can use the
 `--help` flag to get more details on what flags are available. The flag
 `--dump-paths` will print to screen the environment variables `PATH`, `CPATH`,
 `LD_LIBRARY_PATH`, `LIBRARY_PATH`, and `CMAKE_PREFIX_PATH` at link time. The
-flag `--dump-library-versions` will print to screen the contents of the
-`LibraryVersions.txt` file generated by CMake. The flag `--dump-environment`
+flag `--dump-build-info` will print to screen the contents of the
```

Comment:
There's one more `--dump-library-version` in line 99 above

Reply:
Oops, that one has a typo (no final 's') and so I missed it in the find-replace... thanks for noticing!